client/vscode/src/webview/search-panel/RepoView.tsx

```diff
@@ -132,12 +132,12 @@ export const RepoView: React.FunctionComponent<React.PropsWithChildren<RepoViewP
                                     )}
                                 >
                                     <span>
-                                        {entry.isDirectory && (
-                                            <Icon className="mr-1 text-muted" as={FolderOutlineIcon} />
-                                        )}
-                                        {!entry.isDirectory && (
-                                            <Icon className="mr-1 text-muted" as={FileDocumentOutlineIcon} />
-                                        )}
+                                        <Icon
+                                            role="img"
+                                            aria-hidden={true}
```

Comment:
```suggestion
                                            title={entry.isDirectory ? 'Folder' : 'File'}
```

I think this adds value